pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -173,6 +176,10 @@ self: super: {
     dependencies = with super; [ ultisnips ];
   });
 
+  fzf-vim = super.fzf-vim.overrideAttrs(old: {
+    dependencies = with super; [ fzfWrapper ];
```

Comment:
This breaks evaluation with the following message (since `fzfWrapper` is in the same overlay):

```
error: undefined variable 'fzfWrapper' at /home/ma27/Projects/nixpkgs/pkgs/misc/vim-plugins/overrides.nix:180:34
(use '--show-trace' to show detailed location information)
```